lib/components/navigation/SideBar.tsx

```diff
@@ -22,6 +22,8 @@ const SideBar: FC<SideBarProps> = ({ open, onClose }) => {
   }, [onClose, router]);
 
   const { sites } = useSites();
+  const [isEditMode, setEditMode] = useState(false);
+  const [isSelected, setSelected] = useState('');
```

Comment:
I think we can be a bit more clear by setting the type of this to `string | null`, rather than having an empty string correspond to nothing selected

```suggestion
  const [isSelected, setSelected] = useState<string | null>(null);
```